tests/CarbonSleeper/SleepTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Tests\Carbon;
+
+/*
+ * This file is part of the Carbon package.
+ *
+ * (c) Brian Nesbitt <brian@nesbot.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Carbon\Carbon;
+use Carbon\CarbonSleeper;
+use Tests\AbstractTestCase;
+
+class SleepTest extends AbstractTestCase
+{
+    protected function tearDown()
```

Comment:
Interesting point @rrehbeindoi , probably some involvments to add to wrapWithTestNow then. You cna leave it as it for now then